package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stripe/connect-js",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
Don't bump this before publishing! Also this should be a minor version bump as the interface is changing.